### PR TITLE
chg: improve logging

### PR DESCRIPTION
**Pull request by philippseith**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -140,6 +140,7 @@ func (c *client) Start() {
 			// RUN!
 			err := c.run()
 			if err != nil {
+				_ = c.info.Log("error", fmt.Sprintf("%v", err))
```
Comment (andig) [inline client.go]: c.error.Log("connect", ...)

?

**Comment by andig** [comment]
This is what appears in my log:

    [easee ] TRACE 2021/11/22 18:29:03 level=info error=connect level=info error=connect
    [easee ] TRACE 2021/11/22 18:29:04 level=info error=connect level=info error=connect level=info error=connect
    [easee ] TRACE 2021/11/22 18:29:05 level=info error=connect level=info error=connect level=info error=connect level=info error=connect

Seems the errors are concatenated?

**Comment by philippseith** [comment]
Maybe a problem with your logger.
With 
```go
import "GitHub.com/go-kit/log"
...
signalr.Logger(log.NewLogfmtLogger(os.Stdout), false))
```
it looks like this:
```
level=info connect="POST http://localhost:8086/chut/negotiate -> 404 Not Found"
level=info connect="POST http://localhost:8086/chut/negotiate -> 404 Not Found"
level=info connect="POST http://localhost:8086/chut/negotiate -> 404 Not Found"
level=info connect="POST http://localhost:8086/chut/negotiate -> 404 Not Found"
level=info connect="POST http://localhost:8086/chut/negotiate -> 404 Not Found"
```

File: client.go [hunk 2]
```diff
@@ -98,6 +98,12 @@ func NewClient(ctx context.Context, options ...func(Party) error) (Client, error
 			}
 		}
 	}
+	// Wrap logging with timestamps
```
Comment (andig) [inline client.go:101]: could we please remove these again? They produce duplicate timestamps if using any other logging lib. Adding the timestamp should imho be task of the user-chosen logging lib, not of the log-generating code.

Example:

```
[easee ] TRACE 2021/11/22 21:23:39 ts=2021-11-22T20:23:39.681722Z level=info connect=connect
```

**Comment by andig** [comment]
Thanks for the quick fix! Replicated log strings were missing `strings.Builder.Reset()` in my code, thank you. Opened #107 for follow-up.

File: client.go (hunk 2 above)
Comment (philippseith) [reply client.go:101]: IMHO the timestamp should be taken when the event occurs. This is vital if you have a distributed system.
go-kit/log is a structured logger, so it should be simple to strip the ts key/value pair if you like to add a timestamp yourself.

**Pull request merged**